src/react/App.css

```diff
@@ -112,6 +112,14 @@ body {
 .text {
   margin: 1vh 2vh;
   color: black;
+  font-size: large;
+  text-align: center;
+}
+.small-text {
+  margin: 1vh 2vh;
+  color: black;
+  font-size: medium;
+  text-align: left;
```

Comment:
another good practice is to prepend the component name to the css classes, so that there are no unforeseen overwrites and you only use the class in correct places. so here the classes should be `.app__text` (although it seems too generic, what text is it?) and `.app__small-text`

Reply:
I'll rename all the css classes for my next pull request with the colors because it fits more closely with the theme